python/dask_cudf/dask_cudf/core.py

```diff
@@ -98,6 +100,12 @@ def to_dask_dataframe(self, **kwargs):
             M.to_pandas, nullable_pd_dtype=nullable_pd_dtype
         )
 
+    @property
+    def values(self):
+        a = self.map_partitions(methods.values)
+        a._meta = cp.asarray(a._meta)
```

Comment:
Why do we need to do this coercion though? From a quick look `methods.values` should return cupy arrays for cuDF DataFrames so shouldn't the meta be calculated the same way?

Reply:
Good point. Maybe we can fix this in Dask after all 🙂